modules/plugin/coverage-multidim/netcdf/src/main/java/org/geotools/imageio/netcdf/utilities/NetCDFCRSUtilities.java

```diff
@@ -129,7 +129,7 @@
     public static final String CONVERT_AXIS_KM_KEY =
             "org.geotools.coverage.io.netcdf.convertAxis.km";
 
-    private static final boolean CONVERT_AXIS_KM;
+    private static boolean convertAxisKm;
```

Comment:
Just thinking out loud here... if the name is left as CONVERT_AXIS_KM then the existing code in GeoServer would not need to be changed right? This would allow starting to merge the GeoTools part without having to go and also have a GeoServer PR ready (the GeoServer PR for Java 17 would still use the new setter).

Reply:
Not sure that would work because the removal of final

Reply:
Don't have to finish/merge the full GeoServer Java 17 thing, could just merge this specific one in coordination - https://github.com/geoserver/geoserver/pull/5319.